daffodil-runtime1/src/main/java/org/apache/daffodil/api/ValidationMode.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.daffodil.api;
+
+/**
+ * Validation modes for validating the resulting infoset against the DFDL schema
+ */
+public enum ValidationMode {
+  /**
+   * Turn off all validation against the DFDL schema
+   */
+  Off(10),
```

Comment:
Not that I'm against it, but is there a reason we aren't doing the more traditional ALL_CAPS for the enums?

Reply:
Hmm, we ended up getting rid of ValidationMode enums, but I based it off the Scala one which was not all caps iirc

Reply:
I think all of these Java enums play a role in the API as well as in the scala code internally, and the all-caps thing makes for awkward scala code. 

Particularly so for DFDLPrimType as these match the DFDL simple type names with CamelCase.  

So I think there is a pretty good reason for our Java api to use CamelCase enums.